scripts/proxy-test/README.md

```diff
@@ -0,0 +1,9 @@
+This is a simple test to ensure we can make a websocket connection through a proxy server. It sets up a
```

Comment:
Not positive if this is the best place for this directory to live.

Reply:
I like this setup, but it feels like it should live in an integrations tests directory (which doesn't exist). I think it's good here for now and we can move it to a more suitable location when we reorganize our integration tests as a part of https://github.com/PrefectHQ/prefect/issues/16179.